python/lsst/pipe/base/graph/graph.py

```diff
@@ -696,15 +726,11 @@ def _buildSaveObject(self) -> bytearray:
             # compressing has very little impact on saving or load time, but
             # a large impact on on disk size, so it is worth doing
             dump = lzma.compress(pickle.dumps(taskDef, protocol=protocol))
-            taskDefMap[taskDef.label] = (count, count+len(dump))
+            taskDefMap[taskDef.label] = {"bytes": (count, count+len(dump))}
```

Comment:
Is there only one key ("bytes") in this dict? I guess this is to make things more extensible in the future but this also add additional incompatibility between versions 1 and 2. Same question for `nodeMap` below.

Reply:
I am setting this up for another upcoming ticket which I should be able to speed up load times. I wanted to change the format now, instead of having a larger change in version 3 of the file format.

This also plays into some possible future changes if/when QuantumDirectories come into play.